go/api/commands.go

```diff
@@ -0,0 +1,52 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+// StringCommands defines an interface for the "String Commands" group of commands for standalone and cluster clients.
+//
+// See [valkey.io] for details.
+//
+// [valkey.io]: https://valkey.io/commands/?group=string
+type StringCommands interface {
+	// Set the given key with the given value. The return value is a response from Valkey containing the string "OK".
+	//
+	// See [valkey.io] for details.
+	//
+	// For example:
+	//
+	//	result := client.Set("key", "value")
```

Comment:
I think we need to define a standard/template for docs and align all docs with it.
```suggestion
	//	result, err := client.Set("key", "value")
```

Meanwhile it is not such important for now, we can do it later.

Reply:
Done